### ISSUE TITLE
Paginator: Missing aria-current="page" 

### ISSUE THREAD

**Issue by Bukama** [primefaces/primefaces#10741]
### Describe the bug

Our accessibility test showed an issue with the Primefaces [Paginator](https://www.primefaces.org/showcase/ui/data/datatable/paginator.xhtml). The test states that the current selected page should has the [`aria-current="page"` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current).

You can see this also in the [W3C Pagination example](https://design-system.w3.org/components/pagination.html) where "Page 8" has the `aria-current` attribute.

### Reproducer

Primefaces showcase

### Expected behavior

The selected page of the paginator should has the `aria-current="page"` attribute.

### PrimeFaces edition

Community

### PrimeFaces version

13.0.1

### JSF implementation

Mojarra

### JSF version

2.x

### Java version

8

**Comment by Bukama** [comment]
Don't ask me why this wasn't reported before in the three successful tests we already had :(

**Comment by melloware** [comment]
Yes let me investigate.
